tiatoolbox/models/architecture/nuclick.py

```diff
@@ -188,7 +189,8 @@ def __init__(
         strides: Union[Tuple[int, int], np.ndarray] = (1, 1),
         activation: str = "relu",
         use_bias: bool = False,
-    ):
+    ) -> None:
+        """Initializes :class:`MultiscaleConvBlock`."""
```

Comment:
```suggestion
        """Initialize :class:`MultiscaleConvBlock`."""
```

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/625/commits/c2e1cd4ccfd6aaeb32664f23d4f6634cddb90090